storagemarket/provider_test.go

```diff
@@ -1103,6 +1103,61 @@ func TestDealVerification(t *testing.T) {
 	}
 }
 
+func TestIPNIAnnounce(t *testing.T) {
+	ctx := context.Background()
+
+	t.Run("announce", func(t *testing.T) {
+		// setup the provider test harness
+		harness := NewHarness(t)
+		// start the provider test harness
+		harness.Start(t, ctx)
+		defer harness.Stop()
+
+		td := harness.newDealBuilder(t, 1).withAllMinerCallsNonBlocking().withNormalHttpServer().build()
+		td.params.SkipAnnounceToIPNI = false
```

Comment:
Rather than setting `td.params.SkipAnnounceToIPNI` explicitly, I'd suggest we follow the pattern of how the other parameters are set: add a `withAnnouncement(bool)` method to the builder that sets `td.params.SkipAnnounceToIPNI` appropriately

Reply:
Resolved in [be5958e](https://github.com/filecoin-project/boost/pull/1051/commits/be5958e0b657e897f70c73589710d16557cb71ef)